db/silkworm/etl/file_provider.hpp

```diff
@@ -0,0 +1,39 @@
+/*
+   Copyright 2020 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+#ifndef ETL_SILKWORM_FILE_PROVIDER_H
+#define ETL_SILKWORM_FILE_PROVIDER_H
+
+#include <fstream>
+#include <boost/filesystem/operations.hpp>
```

Comment:
Move this include into `file_provider.cpp`. No boost:filesystem's is used here and there's no need to include this in headers which include this one.